icepyx/core/is2ref.py

```diff
@@ -149,14 +149,14 @@ def get_varlist(elem):
 
```

Comment:
Under `_get_custom_options` function, need to update the `dataset` in `capability_url =(f"https://n5eil02u.ecs.nsidc.org/egi/capabilities/{dataset}.{version}.xml")` into `product` as well, otherwise `region_a.show_custom_options(dictview=True)` will throw a `NameError: name 'dataset' is not defined`.